libs/genai/langchain_google_genai/chat_models.py

```diff
@@ -455,6 +455,27 @@ def _parse_response_candidate(
             elif text:
                 raise Exception("Unexpected content type")
 
+        if part.inline_data.mime_type.startswith("image/"):
+            import base64
+
+            base64_data = base64.b64encode(part.inline_data.data).decode("utf-8")
```

Comment:
can we use `_image_utils.image_bytes_to_b64_string` instead, please?

Reply:
sure! update.